Feature/rdk 56050

dp0000 opened the pull request:
rebasing pull request to dev_sprint_25_2

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: Migration to dev_sprint_25_2 from 117308 (patchset 12)
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: Migration to dev_sprint_25_2 from 117308 (patchset 12)
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

dp0000 committed:
Delete AampConfig.cpp.rej

removed .rej

dp0000 committed:
Delete aampgstplayer.cpp.rej

dp0000 committed:
Delete CMakeLists.txt.rej

Deepikasri1234 committed:
Merge branch 'feature/RDK-56050' of https://github.com/rdkcentral/aamp into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
Merge remote-tracking branch 'origin/dev_sprint_25_2' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
Merge branch 'develop' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=204) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=204) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: dp0000<br/>:x: Deepikasri1234<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=204) it.</sub>

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 9 **violation**s, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/aamp/204/rdk/components/generic/aamp

* Commit: 05b184593795d26f3b9248acf5985bb46d4fb00c

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/79dc59e2ef34de5af1198fce8300fe3c)'

Deepikasri1234 committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-56050

rdkcmf-jenkins commented:
b'## **WARNING:** A Copyright scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: 92b4683a588bba93109353d3db6665006f282cb8
'

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 9 **violation**s, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/aamp/204/rdk/components/generic/aamp

* Commit: e0a088bc1bf6d0c6aed950a970ab16cec0865180

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/5eada2cc9d5943d72efceb4f736e10c7)'

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** bf3fd751844d7236ddcf0a06b5eef95f9d7c748e
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/1e430d1638ef2b3e3736292f912b014f'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 9 **violation**s, 0 files pending approval, 0 files pending identification.
* Protex Server Path: /home/blackduck/github/aamp/204/rdk/components/generic/aamp

* Commit: bf3fd751844d7236ddcf0a06b5eef95f9d7c748e

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/804d0ee161e8731d14c7f4588c848311)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Copyright scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: bf3fd751844d7236ddcf0a06b5eef95f9d7c748e
'
